doc/articles/features/pointers-keyboard-and-other-user-inputs.md

```diff
@@ -0,0 +1,183 @@
+# Uno support for user inputs
```

Comment:
Great doc! Should be added to TOC https://github.com/unoplatform/uno/blob/master/doc/articles/toc.yml